src/components/common/Filter.tsx

```diff
@@ -73,9 +73,9 @@ export default function Filter({
   if (!open) return null;
 
   return (
-    <div className="z-5 flex w-full max-w-390 flex-col gap-24 rounded-[10px] border border-gray-20 bg-white px-20 py-24 shadow-custom">
-      <div className="flex flex-row items-center justify-between text-[2rem] leading-[2.4rem]">
-        <h3 className="text-h3 font-bold">상세 필터</h3>
+    <div className="z-5 flex w-full max-w-390 flex-col gap-24 rounded-xl border border-gray-20 bg-white px-12 py-24 shadow-custom md:px-20">
```

Comment:
💬 border 때문에 1px 줄여줘야 맞지 않을까 싶네요~
```suggestion
    <div className="z-5 flex w-full max-w-390 flex-col gap-24 rounded-xl border border-gray-20 bg-white px-12 py-24 shadow-custom md:px-19">
```